.gitignore

```diff
@@ -6,7 +6,9 @@
 !/cmake/*.cmake
 *~
 /test/benchmark_test
+/test/benchmark_test.exe
```

Comment:
Some people do build in-tree, though, even if it's not the best idea...

Me, I'd make a different comment: this should be a ```*.exe``` entry, the same way object files are handled.